api/alarms/index.js

```diff
@@ -0,0 +1,182 @@
+let display = document.querySelector('.alarm-display');
```

Comment:
I know this is stylistic but I really prefer these to be `const`. They'll never change and it makes it much clearer looking at the code. I don't need you to go to all PRs and change this, just some advice.

Reply:
I'm amenable to that. This is just a stylistic thing I haven't adopted. Please ping me in the future if you see lapses.